test/test_constraints_file.py

```diff
@@ -156,7 +161,7 @@ def test_verify_working_days_for_employee_does_not_exceed_the_max_working_days()
     shifts: list[Shift] = []
 
     for _ in range(0, 2):
-        employees.append(Employee(f"{_}", EmployeePriorityEnum.HIGHEST, EmployeeStatusEnum.senior_employee, employee_id=uuid4(), position=EmployeePositionEnum.part_timer))
+        employees.append(Employee(f"{_}", EmployeePriorityEnum.HIGHEST, EmployeeStatusEnum.senior_employee, employee_id=uuid4(), position=EmployeePositionEnum.part_timer, preferences=Preferences([], [], [])))
```

Comment:
it bothers me that one needs to define everything, you can make dataclasses have default constructors